adminSiteClient/gdocsValidation.ts

```diff
@@ -106,6 +111,36 @@ export class ExcerptHandler extends AbstractHandler {
     }
 }
 
+export class ContentHandler extends AbstractHandler {
+    handle(gdoc: OwidArticleType, messages: ErrorMessage[]) {
+        gdoc.content.body?.map((block) => {
+            recursivelyMapArticleBlock(block, (block) => {
+                block.parseErrors.forEach((parseError) => {
+                    messages.push({
+                        property: "body",
+                        type: parseError.isWarning
+                            ? ErrorMessageType.Warning
+                            : ErrorMessageType.Error,
+                        message: parseError.message,
+                    })
+                })
+                // For now, only images can have data errors
+                if (block.type === "image") {
+                    block.dataErrors.forEach((dataError) => {
```

Comment:
I wanted a separate interface to store errors that aren't to do with parsing the archie syntax - in this case, if something's wrong with the image file that's being referenced.